stdlib/knative/src/main/ballerina/src/knative/annotation.bal

```diff
@@ -0,0 +1,250 @@
+# + dockerCertPath - Docker certificate path. Default is "DOCKER_CERT_PATH" environment variable.
+# + registry - Docker registry url.
+# + username - Username for docker registry.
+# + password - Password for docker registry.
+# + baseImage - Base image for docker image building. Default value is `"ballerina/ballerina-runtime:<BALLERINA_VERSION>"`.
+# Use `"ballerina/ballerina-runtime:latest"` to use the latest stable ballerina runtime docker image.
+# + image - Docker image name with tag. Default is `"<OUTPUT_FILE_NAME>:latest"`. If field `registry` is set then it will be prepended to the docker image name as `<registry>/<OUTPUT_FILE_NAME>:latest`.
+# + buildImage - Docker image to be build or not. Default is `true`.
+# + push - Enable pushing docker image to registry. Field `buildImage` must be set to `true` to be effective. Default value is `false`.
+# + copyFiles - Array of [External files](kubernetes#FileConfig) for docker image.
+# + singleYAML - Generate a single yaml file with all kubernetes artifacts (services, deployment, ingress and etc). Default is `true`.
+# + namespace - Kubernetes namespace to be used on all artifacts.
+# + replicas - Number of replicas. Default is `1`.
+# + livenessProbe - Enable/Disable liveness probe and configure it. Default is `false`.
+# + readinessProbe - Enable/Disable readiness probe and configure it. Default is `false`.
+# + imagePullPolicy - Image pull policy. Default is `"IfNotPresent"`.
+# + env - Environment variable map for containers.
+# + podAnnotations - Map of annotations for pods.
+# + podTolerations - Toleration for pods.
+# + dependsOn - Services this deployment depends on.
+# + imagePullSecrets - Image pull secrets.
+# + containerConcurrency - concurent request handle by one container instance
+# + timeoutSeconds - max time the instance is allowed for responding to a request
+# + port - containerPort value for Knative service
+public type ServiceConfiguration record{|
+        *Metadata;
+        string dockerHost?;
+        string dockerCertPath?;
+        string registry?;
+        string username?;
+        string password?;
+        string baseImage?;
+        string image?;
+        boolean buildImage = true;
+        boolean push = false;
+        FileConfig[] copyFiles?;
+        boolean singleYAML = true;
+        string namespace?;
+        int replicas = 1;
+        boolean|ProbeConfiguration livenessProbe = false;
+        boolean|ProbeConfiguration readinessProbe = false;
+        ImagePullPolicy imagePullPolicy = IMAGE_PULL_POLICY_IF_NOT_PRESENT;
+        map<string|FieldRef|SecretKeyRef|ResourceFieldRef|ConfigMapKeyRef> env?;
+        map<string> podAnnotations?;
+        PodTolerationConfiguration[] podTolerations?;
+        string[] dependsOn?;
+        string[] imagePullSecrets?;
+        int containerConcurrency=100;
+        int timeoutSeconds=60;
+        int port=8080;
```

Comment:
Missing space around the `=` sign. Reformat the file.

Reply:
changes added